### ISSUE TITLE
Tests fail (with Function::Parameters 2.x?)

### ISSUE THREAD

**Issue by eserte** [zurborg/libmoove-perl#1]
The test suite started to fail on my smokers:
```
Use of uninitialized value $package in hash element at /home/cpansand/.cpan/build/2017042206/Moove-0.004-0/blib/lib/Moove.pm line 100.
Compilation failed in require at t/all.t line 5.
BEGIN failed--compilation aborted at t/all.t line 5.
# Looks like your test exited with 255 before it could output anything.
t/all.t ................... 
Dubious, test returned 255 (wstat 65280, 0xff00)
Failed 2/2 subtests 
...
Bailout called.  Further testing stopped:  
#   Failed test 'use t::TryCatch;'
#   at t/trycatch.t line 7.
#     Tried to use 't::TryCatch'.
#     Error:  In func test1: I was expecting a parameter list, not "{" at t/TryCatch.pm line 5.
# Compilation failed in require at t/trycatch.t line 7.
# BEGIN failed--compilation aborted at t/trycatch.t line 7.
FAILED--Further testing stopped.
```

It seems that this happens with Function::Parameters 2.x (@mauke: FYI). Statistical analysis:
```
****************************************************************
Regression 'mod:Function::Parameters'
****************************************************************
Name           	       Theta	      StdErr	 T-stat
[0='const']    	      1.0000	      0.0000	95261290292804176.00
[1='eq_1.0706']	      0.0000	      0.0000	   0.82
[2='eq_2.000003']	     -1.0000	      0.0000	-36378506130494768.00
[3='eq_2.000006']	     -1.0000	      0.0000	-45328270394558880.00

R^2= 1.000, N= 61, K= 4
****************************************************************
```

**Comment by zurborg** [comment]
Yes, that seems to be broken with F::P 2. I'll have a look at it, and thanks to @mauke for the fix!

**Comment by zurborg** [comment]
Fixed in v0.005 release.
